toolkit/tools/scheduler/schedulerutils/preparerequest.go

```diff
@@ -75,6 +75,13 @@ func canUseCacheForNode(pkgGraph *pkggraph.PkgGraph, node *pkggraph.PkgNode, pac
 		return
 	}
 
+	// If delta build enabled, then we can always use the cache
```

Comment:
For future reference, maybe add a bit to the end such as "unless a dependency of this package is rebuilding" or similar. (ie pkgA -> pkgB -> pkgC, if pkgC is rebuilding both A and B will also be rebuilt)